components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -343,7 +374,7 @@ open class GleanInternalAPI internal constructor () {
      * @param pingNames List of pings to send.
      * @return true if any pings were actually sent.
      */
-    internal fun sendPings(pingNames: List<String>): Boolean {
+    internal fun sendPingsInternal(pingNames: List<String>): Boolean {
```

Comment:
nit: this is not strictly related to this patch, but if consumers will start attempting to send empty pings using the `sendPings` API they will receive no feedback if the payload is empty and nothing happens. I'd suggest adding a debug log line [here](https://github.com/mozilla-mobile/android-components/blob/0dfbad94ab623d922d1f9b4d41be6bc594e80f63/components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt#L392) to just say "Nothing to collect for ping: `pingname`".

Reply:
That's not a bad idea.  This information is currently in the return value, too.  Do you think maybe that's not necessary?  None of the uses internally use the return value right now.

Reply:
Mh, I'm not sure if that's needed or not. I might have a use case for the `activation` ping, so let's keep it around for now :)